backend/apps/events/apis.py

```diff
@@ -117,3 +120,21 @@ def patch(self, request, event_id):
         data = EventDetailApi.OutputSerializer(event).data
 
         return Response(data)
+
+class EventTicketCreateApi(APIView):
+    class OutputSerializer(serializers.ModelSerializer):
+        class Meta:
+            model = EventTicket
+            fields = ['id', 'ticket_id', 'qr_code', 'event', 'user']
+
+    def post(self, request, event_id):
+        user = request.user
+        try:
+            event = Event.objects.get(id=event_id)
+        except Event.DoesNotExist:
+            raise Http404
+
+        ticket = event_ticket_create(event=event, user=user)
+
+        serializer = self.OutputSerializer(ticket)
+        return Response(serializer.data, status=status.HTTP_201_CREATED)
```

Comment:
where is the update endpoint? it should allow for event organisers to send an update request for a ticket to mark it as used